etc/calico/confd/templates/bird.cfg.template

```diff
@@ -166,6 +166,9 @@ template bgp bgp_template {
 {{if ls "/global/peer_v4"}}
 {{range gets "/global/peer_v4/*"}}{{$data := json .Value}}
 {{$nums := split $data.ip "."}}{{$id := join $nums "_"}}
+{{- if $data.port}}
+{{- $id = printf "%s_%.0f" $id $data.port}}
```

Comment:
IS `%d` (for decimal) not available; seems odd to use a float format

Reply:
`%d` was the initial choice, say if port is 179 it prints `%!d(float64=179)`, that's reason for using `%f`